keyboards/mechlovin/hannah65/rev1/rules.mk

```diff
@@ -0,0 +1 @@
+DEFAULT_FOLDER = mechlovin/hannah65/rev1/haus
```

Comment:
I'm really fond of this, as it changes the outcome of `qmk compile -kb mechlovin/hannah65/rev1 -km default`.